crates/forge-runner/src/build_trace_data.rs

```diff
@@ -273,6 +272,10 @@ fn build_profiler_deprecated_syscall_selector(
         DeprecatedSyscallSelector::Sha256ProcessBlock => {
             ProfilerDeprecatedSyscallSelector::Sha256ProcessBlock
         }
+        DeprecatedSyscallSelector::GetClassHashAt => {
+            ProfilerDeprecatedSyscallSelector::GetClassHashAt
+        }
+        DeprecatedSyscallSelector::KeccakRound => unimplemented!("No corresponding syscall"),
```

Comment:
@THenry14 this might be relevant for profiling

Reply:
yup :+1: should be added to cairo annotations. You can do it straight away; otherwise I will add this in a next version alongside sierra gas additions